<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1661</width>
    <height>1411</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>10</y>
     <width>491</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Asetukset</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tuotekuvakansio</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kameran numero</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="productPicFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Kansio, johon tuotekuvat tallennetaan. Kansio sijaitsee käyttäjän Kuvat-kansiossa. Tallentuu asetuksiin.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vaakaresoluutio</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pystyresoluutio</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="hResolution">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Kameran maksimi vaakaresoluutio (kuvan leveys) pikseleinä. Tallentuu asetuksiin&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>640</number>
    </property>
    <property name="maximum">
     <number>3840</number>
    </property>
    <property name="singleStep">
     <number>320</number>
    </property>
   </widget>
   <widget class="QPushButton" name="videoReset">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pysäyttää videokameran&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Reset Video</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="vResolution">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Kameran suurin pystyresoluutio (kuvan korkeus) pikseleinä. Tallentuu asetuksiin.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>480</number>
    </property>
    <property name="maximum">
     <number>2160</number>
    </property>
    <property name="singleStep">
     <number>240</number>
    </property>
    <property name="value">
     <number>480</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="cameraIxSpinBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Videokameran numero, tallentuu asetuksiin&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>501</height>
    </rect>
   </property>
   <property name="title">
    <string>Tuotekuvan ottaminen</string>
   </property>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>601</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>150</y>
      <width>20</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="productId">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tuotteen koodi: oma tai valmistajan EAN-koodi&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="viewFinderRect">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>601</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>20</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="tuotekoodiLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>29</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tuotekoodi</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>601</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="productImage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Videokuva tuotteesta, sovita tuotteen koko siten, että se täyttää koko kehyksen. Nosta kameraa ylöspäin suuria esineitä kuvattaessa ja laske alaspäin pieniä esineitä kuvattaessa.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="videokuvaLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Videokuva</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nimike</string>
    </property>
   </widget>
   <zorder>productImage</zorder>
   <zorder>line_2</zorder>
   <zorder>line_4</zorder>
   <zorder>productId</zorder>
   <zorder>viewFinderRect</zorder>
   <zorder>line_3</zorder>
   <zorder>tuotekoodiLabel</zorder>
   <zorder>line</zorder>
   <zorder>videokuvaLabel</zorder>
   <zorder>lineEdit</zorder>
   <zorder>label_7</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>270</y>
     <width>491</width>
     <height>511</height>
    </rect>
   </property>
   <property name="title">
    <string>Istunnossa tallennetut tuotekoodit</string>
   </property>
   <widget class="QListWidget" name="pictureList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>471</width>
      <height>481</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="stillButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>520</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ottaa yksittäiskuvan (still) tuotteesta ja näyttää sen esikatselussa.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Ota Kuva</string>
   </property>
  </widget>
  <widget class="QPushButton" name="captureButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Käynnistää videokameran Asetukset ryhmässä näkyvin asetuksin&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Käynnistä Video</string>
   </property>
  </widget>
  <widget class="QPushButton" name="printCardButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>520</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tulostaa tuotekuvasivun tuoteluetteloa varten&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Luo PDF-tiedosto</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="productCard">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>411</width>
     <height>771</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="catalogPicture">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>360</width>
      <height>640</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="bcLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>710</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Libre Barcode 128</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Tuotekuvaston viivakoodi pienennettynä</string>
    </property>
    <property name="text">
     <string>Viivakoodi</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>productId</tabstop>
  <tabstop>captureButton</tabstop>
  <tabstop>stillButton</tabstop>
  <tabstop>productPicFolder</tabstop>
  <tabstop>cameraIxSpinBox</tabstop>
  <tabstop>videoReset</tabstop>
  <tabstop>hResolution</tabstop>
  <tabstop>vResolution</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>label_6</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>730</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>productId</sender>
   <signal>textChanged(QString)</signal>
   <receiver>bcLabel</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>832</x>
     <y>746</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
